scripts/get_used_routes.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
```

Comment:
il est préférable d'utiliser l'environnement pour déterminer le chemin de `bash`

```suggestion
#!/usr/bin/env bash
```